scripts/jenkins/cloud/ansible/roles/setup_zypper_repos/tasks/setup_maint_update_repos.yml

```diff
@@ -42,44 +42,81 @@
   delay: 5
   until: mu_url is succeeded
 
-- name: Ensure MU directories exists
-  file:
-    state: directory
-    path: "{{ local_repos_base_dir }}/{{ item }}"
-  loop: "{{ maint_updates_list }}"
+# Download MU repo contents and add/enabled/disable MU repos
```

Comment:
I wonder if it is possible to have the same strategy you used on `extra_repos` by having remove/add repositories tasks in different files and including the files dynamically on the role `main.yml`.

I think it would be cleaner as the whole set of tasks would/wouldn't be included based on `zypper_repo_state`

Reply:
So that kinda comes back to why I created SOC-10909; I had originally intending to fix this stuff up under that ticket first, and then land this patch set on top of the result...

However after the re-org and some priority changes I put that ticket back in the backlog, and @rmaliska has taken it up...

So the question then becomes whether we want to fix the MU repo setup stuff here, or leave it as a separate work item?

Reply:
This is more like a suggestion, I am ok with both options.